src/Dropdown.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
react/prop-types should definitely be disabled on the project level (i.e. in `package.json`), but the other one doesn't need to be disabled.